.circleci/config.yml

```diff
@@ -333,18 +333,17 @@ jobs:
           name: Install NPM dependencies
           command: |
             npm --version
+            npm set progress false
             npm install
             npm dedupe
       - save_cache:
-          key: v1-npm-{{ arch }}-{{ .Branch }}-{{ checksum "package.json" }}
+          key: node-v1-{{ .Branch }}-{{ checksum "package-lock.json" }}
```

Comment:
Can't we remove the `{{ .Branch }}` so it is shared by all builds
Or is it dangerous:?